python/tutorials/09-experimental-tma-matrix-multiplication.py

```diff
@@ -191,5 +191,10 @@ def perf(ms):
     return perf(ms), perf(max_ms), perf(min_ms)
 
 
+if torch.cuda.get_device_capability()[0] < 9:
+    import sys
+    print("Skipping TMA benchmark for GPU with compute capability < 9")
+    sys.exit(0)
+
```

Comment:
Block pointer should work on all targets, why we do we need to skip?

Reply:
I think it's related to 

```
{'BLOCK_SIZE_M': 128, 'BLOCK_SIZE_N': 128, 'BLOCK_SIZE_K': 64, 'GROUP_SIZE_M': 8}, num_stages=7, num_warps=4
```

On Ampere, you won't be able to use that large shared memory.
But my point is that if we fallback to a version that works for Ampere, we would get something equivalent as tutorial-08